plugins/dependencies/src/main/kotlin/dependencies/Touchlab.kt

```diff
@@ -24,5 +27,34 @@ object Touchlab : IsNotADependency {
         val isolate = module("stately-isolate")
         val isoCollections = module("stately-iso-collections")
     }
+
+    val kermit = Kermit
+
+    /**
+     * Kermit is a Kotlin Multiplatform logging utility with composable log outputs.
+     * The library provides prebuilt loggers for outputting to platform logging tools
+     * such as Logcat and NSLog.
+     *
+     * [GitHub Releases](https://github.com/touchlab/Kermit/releases)
+     *
+     * Github repo: [touchlab/Kermit](https://github.com/touchlab/Kermit)
+     *
+     * [1.0 release blog](https://medium.com/@kpgalligan/kermit-kmp-logging-1-0-196bf799b738)
+     */
+    object Kermit : DependencyGroup(
+        group = "co.touchlab",
+        rawRule = """
+        co.touchlab:kermit(-*)
+                    ^^^^^^
+        """.trimIndent()
+    ) {
+        val kermit = module("kermit")
```

Comment:
FYI, you could have replaced that `val kermit` that leads to `Touchlab.kermit.kermit` at use-site with making the `Kermit` object extend `DependencyNotationAndGroup`, providing the `name`, which allows to write `Touchlab.kermit` at use-site. I made the change on main.